src/app/shared/pipes/find-by-name.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'findByName'
+})
+export class FindByNamePipe implements PipeTransform {
```

Comment:
The pipe name is "FindByName" and you filter by "title", this is kinda confusing. Rename the pipe name to "find-by-title" or "find-by" and put a first param "findBy". So you will have a pipe where you can pass as parameter what field to be filtered.